org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/entities/EntitiesDocumentationUtils.java

```diff
@@ -0,0 +1,83 @@
+/**
+ *  Copyright (c) 2020 Red Hat, Inc. and others.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v2.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ *  Contributors:
+ *  Red Hat Inc. - initial API and implementation
+ */
+package org.eclipse.lemminx.extensions.entities;
+
+import org.eclipse.lsp4j.MarkupContent;
+import org.eclipse.lsp4j.MarkupKind;
+
+/**
+ * Utilities class to generate entities documentation.
+ *
+ */
+public class EntitiesDocumentationUtils {
+
+	public static enum PredefinedEntity {
+
+		lt("&#38;#60;"), gt("&#62;"), amp("&#38;#38;"), apos("&#39;"), quot("&#34;");

```

Comment:
Should it be `lt("&#60;")` and `amp("&#38;")` ?

Reply:
I copied/pasted those vales from https://www.w3.org/TR/xml/#sec-predefined-ent but I agree withyou, I change it. Thanks!